data/successors/successor news.txt

```diff
@@ -0,0 +1,241 @@
+# Copyright (c) 2024 by Daeridanii
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+news "high houses scion"
+	to show
+		has "language: Successor"
+	location
+		attributes successor
+		attributes wealthy
+		not attributes uninhabited
+	name
+		word
+			"High Houses Scion"
+			"Child of the High Houses"
+			"Scion of the High Houses"
+	message
```

Comment:
Normally what happens is that there is a quotation mark at the beginning and end in separate word blocks, so you may want to do that here.